Makefile

```diff
@@ -46,6 +46,6 @@ build:
 	cd src && make build
 
 clean:
-	cd src && make clean
+	[ -r src/Makefile ] && ( cd src && make clean ) || true
```

Comment:
Oh, probably-- didn't know that. Is it a GNUism or a standard Make thing?
(Do we care? :)